server/api/models/member.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  attributes: {
+    id: {
+      type: 'string',
```

Comment:
is there a reason why id type is string instead of number?

Reply:
actually, on top of that, do we need an id column? isn't it already autogenerated by sails in server/config/models.js?

Reply:
oh i didnt know its autogenerated according to that. ok will remove it then

Reply:
id type is string because ObjectId generated by mongodb is string in sails